statsmodels/discrete/tests/test_discrete.py

```diff
@@ -49,6 +49,45 @@
 DECIMAL_0 = 0
 
 
+def check_jac(self):
+    # moved from CheckModelResults
```

Comment:
why moved?  If it takes `self`, this should probably be a method in a mixin class